File: BibliotecaUFABC/app/src/main/java/com/nintersoft/bibliotecaufabc/book_search_model/BookSearchContract.java
package com.nintersoft.bibliotecaufabc.book_search_model;

import android.provider.BaseColumns;

public class BookSearchContract {
    public static final String DB_NAME = "search.db";
    static final int DB_VERSION = 1;

    private BookSearchContract(){}

    static class BookRenewalItems implements BaseColumns {
        static final String TABLE_NAME = "search_items";
        static final String _ID = "_id";
        static final String COLUMN_NAME_TYPE = "type";
        static final String COLUMN_NAME_CODE = "code";
        static final String COLUMN_NAME_TITLE = "title";
        static final String COLUMN_NAME_AUTHOR = "author";
        static final String COLUMN_NAME_SECTION = "section";
    }
}

File: BibliotecaUFABC/app/src/main/java/com/nintersoft/bibliotecaufabc/model/search/BookSearchContract.kt
package com.nintersoft.bibliotecaufabc.model.search

import android.provider.BaseColumns

object BookSearchContract : BaseColumns {
    const val TABLE_NAME = "ent_BookSearch"

    const val ID = "_id"
    const val COLUMN_NAME_TYPE = "type"
    const val COLUMN_NAME_CODE = "code"
    const val COLUMN_NAME_TITLE = "title"
    const val COLUMN_NAME_AUTHOR = "author"
    const val COLUMN_NAME_SECTION = "section"
    const val COLUMN_NAME_FAVOURITE = "favourite"
}